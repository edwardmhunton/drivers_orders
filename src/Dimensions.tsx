import React from "react";

interface Dimensions {
  width: string;
  height: string;
  length: string;
}

export const Dimensions = (dimensions: Dimensions) => {
  return (
    <div>
      <div>Package Dimensions</div>
      <div>{`W: ${dimensions.width}mm`}</div>
      <div>{`H: ${dimensions.height}mm`}</div>
      <div>{`L: ${dimensions.length}mm`}</div>
    </div>
  );
};
