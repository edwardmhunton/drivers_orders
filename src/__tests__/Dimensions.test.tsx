import React from "react";
import renderer from "react-test-renderer";

import { Dimensions } from "../Dimensions";

describe("the fist test", () => {
  it("Renders Correctly", () => {
    const DimensionsComponent = renderer.create(<Dimensions />).toJSON();
    expect(DimensionsComponent).toMatchSnapshot();
  });
});
