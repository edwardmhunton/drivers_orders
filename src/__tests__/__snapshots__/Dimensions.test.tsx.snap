// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dimensions test Renders Correctly 1`] = `
<div>
  <div>
    Package Dimensions
  </div>
  <div>
    W: undefinedmm
  </div>
  <div>
    H: undefinedmm
  </div>
  <div>
    L: undefinedmm
  </div>
</div>
`;
